### PR TITLE
#19030 | 22 Meta descriptions for dotnet framework

### PR DESCRIPTION
**Pull request by LauraKellerGitHub**
User Story
https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1657392 
Meta descriptions for dotnet/framework/wpf articles.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=19030) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=19030) before we can accept your contribution.<br/><br/>:x: LauraKellerGitHub [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=19030)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=19030) it.</sub>

**Comment by LauraKellerGitHub** [comment]
In progress. Not ready for review.

**Comment by Thraka** [comment]
I changed this to a draft PR. Flip it back when you're ready 😄

**Comment by LauraKellerGitHub** [comment]
@thraka
Meta descriptions ready for review, and feedback or approval.
Thanks!

File: docs/framework/wpf/graphics-multimedia/how-to-animate-a-property-by-using-a-storyboard.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Animate a Property by Using a Storyboard"
+description: Enliven presentations with animations and storyboards for properties in Windows Presentation Foundation (WPF).
```
Comment (adegeo) [inline docs/framework/wpf/graphics-multimedia/how-to-animate-a-property-by-using-a-storyboard.md:3]:
```suggestion
description: Enliven your user interface with animations and storyboards for properties in Windows Presentation Foundation (WPF). 
```

File: docs/framework/wpf/graphics-multimedia/shapes-and-basic-drawing-in-wpf-overview.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Shapes and basic drawing overview
+description: Enhance your presentations with ready-to-use shapes and several layers of rendering services in Windows Presentation Foundation (WPF).
```
Comment (adegeo) [inline docs/framework/wpf/graphics-multimedia/shapes-and-basic-drawing-in-wpf-overview.md:3]:
```suggestion
description: Enhance your user interface with ready-to-use shapes and several layers of rendering services in Windows Presentation Foundation (WPF).
```

**Review by adegeo — APPROVED** [review]
Two minor changes. I don't like saying that you're enhancing a "presentation", it makes it sound like this is powerpoint or something.

**Comment by LauraKellerGitHub** [comment]
@adegeo 
Corrections complete.

**Comment by adegeo** [comment]
Thanks!

**Pull request merged**